src/App.jsx

```diff
@@ -14,70 +16,83 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const goodsWithId = goods.map((good, i) => (
+  {
+    id: i,
```

Comment:
👍 
You can also use some tool like uuid to generate a more unique id.
https://www.npmjs.com/package/uuid
But in this task it's not critical